### ISSUE TITLE
关于分类损失计算中，将torch.nn.CrossEntropyLoss()换成torchvision.ops.sigmoid_focal_loss()的问题

### ISSUE THREAD

**Issue by chenyuuu0705** [WZMIAOMIAO/deep-learning-for-image-processing#602]
你好.
我想将classification中的loss function从CrossEntropyLoss换成Focal Loss,但是遇到了如下问题，input和targets的size不一样，求助，这个地方我该怎么调整。
![image](https://user-images.githubusercontent.com/80811609/180632544-434d8068-e5d2-4fbb-9032-dbc137140187.png)
![image](https://user-images.githubusercontent.com/80811609/180632560-e58cda5c-02d5-478d-a73d-d07989857161.png)

**Comment by WZMIAOMIAO** [comment]
自己改代码的问题，自己看官方文档解决吧，这里不做回答，谢谢。
